app/routes/docs.producers.schema-documentation.mdx

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
Rename this file to `docs.producers.schema.mdx`. (The `-documentation` bit is redundant, because this is a sub route of docs.)